AnnoyancesFilter/Popups/sections/popups_general.txt

```diff
@@ -124,3 +124,4 @@
 ! NOTE: Generic rules end ⬆️
 ! !SECTION: Popups - Generic rules
 !
+||vpd.fi^$popup
```

Comment:
It's a subscription popup, but that rule can't help there. Take a look at the html code. As a hint, we have a solution for this in `subscriptions_specific.txt`.